playbooks/roles/sit.cephfs/tasks/setup/main.yml

```diff
@@ -7,6 +7,12 @@
       - ansible-collection-community-general
     state: latest
 
+- name: Install samba-vfs-cephfs to support features like vfs_cephs_snapshots
+  yum:
+    name:
+      - samba-vfs-cephfs
+    state: latest
+
```

Comment:
These tasks are included from _setup.prep_ role which is supposed to run only on [admin] nodes. You'll most probably want to install samba-vfs-cephfs on storage nodes.